lib/clientv3/profile.go

```diff
@@ -41,6 +44,14 @@ type profiles struct {
 // Create takes the representation of a Profile and creates it.  Returns the stored
 // representation of the Profile, and an error, if there is any.
 func (r profiles) Create(ctx context.Context, res *apiv3.Profile, opts options.SetOptions) (*apiv3.Profile, error) {
+	if res.Name == cresources.DefaultAllowProfileName {
+		return nil, cerrors.ErrorOperationNotSupported{
```

Comment:
Hm, I wonder if the "right" thing to do here is to return an "AlreadyExists" error, rather than a "NotSupoprted". 

I think probably so - this would be the expected behavior from the client's perspective. NotSupported is probably appropriate for Update/Delete, though.

Reply:
d3c9ca2133f28c52cbaac10d658a252057e0adb1